layouts/partials/tw/individual-newsletter-form.html

```diff
@@ -0,0 +1,52 @@
+{{ $gaForm := .Param "gaForm" }}
+{{ $gaCategory := .Param "gaCategory" }}
+{{ $hiddenField := .Param "hiddenField" }}
+{{ $cta := .Param "cta" }}
+
+{{ $idEmail := partial "helper/new-id" . }}
+
+
+<form
+  class="mt-4 font-sans text-black"
+  x-data="newsletter($el.dataset.baseUrl)"
+  data-base-url="{{ page.Param `newsletters-base-url` }}"
+  @submit.prevent="submit"
+  data-ga-category="{{ $gaCategory }}"
+>
+  {{ partial $hiddenField . }}
+
+
+  <div
+    class="flex h-[44px] w-full items-center justify-between rounded-md border-[1px] border-[#CFCFCF] bg-white px-[20px]"
+  >
+    <input
+      id="{{ $idEmail }}"
+      class="w-full bg-white font-bold text-g-8 placeholder-g-6 focus:outline-none"
+      type="email"
+      name="EMAIL"
+      placeholder="Email address"
+      required
+    />
+
+    <svg aria-hidden="true" class="ml-2 h-[20px] w-[20px] fill-[#CFCFCF]">
```

Comment:
Instead of specifying fill directly, I usually use `fill-current` and then specify the color, like `text-s-4`.